labdao/plex
# bool input supported in tools

## What type of PR is this? 
- 🎮 Feature
- 🐛 Bug Fix

## Description

This allows Boolean support on the frontend (it was already supported on the backend).

<img width="818" alt="Screenshot 2024-02-09 at 8 56 01 AM 1" src="https://github.com/labdao/plex/assets/9427089/ff94ddce-6d65-415e-a535-f85cf51b2ea0">

## Related Tickets & Documents

[Ticket](https://linear.app/convexitylabs/issue/LAB-1179/boolean-support-in-labsay)

## Steps to Test

Run LabSay with the Speedup Boolean True and watch the logs to see the faster printout. 

## Relevant GIF 

<img width="432" alt="Screenshot 2024-02-09 at 8 58 24 AM" src="https://github.com/labdao/plex/assets/9427089/6c6189fb-491d-4d82-acbc-94bca1e4adf1">
